.eslintrc.js

```diff
@@ -3,12 +3,15 @@ module.exports = {
     "**/node_modules/**/*",
     "**/dist/**/*",
   ],
+  settings: {
+    react: {
+      version: "16.14.0",
```

Comment:
perhaps `"version": "detect"`?

Reply:
just in case we upgrade React in the future, or you can import react version from package.json and use here.